etl/steps/data/meadow/oecd/2016-06-01/regional_wellbeing.py

```diff
@@ -0,0 +1,122 @@
+# # Regional wellbeing (OECD 2016)
+
+# +
+import pandas as pd
+
+from owid.catalog import Dataset, Table
+from owid.walden import Catalog as WaldenCatalog
+from etl.steps.data.converters import convert_walden_metadata
+
+# -
+
+# ## Load the data
+
+walden_ds = WaldenCatalog().find_one("oecd", "2016-06-01", "regional_wellbeing")
+df = pd.read_csv(walden_ds.local_path)
+
+# ## Drop useless columns
+
+# some columns are empty
+df.dropna(how="all", axis=1, inplace=True)
+
+# some columns have only a single value for the whole table -- not informative
+for col in df.columns:
+    if len(df[col].unique()) == 1 and col != "Time":
+        df.drop(col, axis=1, inplace=True)
+
+# ## Standardise names
+
+df.columns = [
+    "region_code",
+    "region_name",
+    "ind_code",
+    "ind_name",
+    "year",
+    "unit_code",
+    "unit_name",
+    "value",
+]
+
+df["ind_code"] = df.ind_code.apply(lambda s: s.lower())
+
+# ## Sanity checks
+
+# #### One unit per indicator
+
+assert len(df[["ind_code", "unit_code"]].drop_duplicates()) == len(df.ind_code.unique())
+
+# #### One name per indicator
+
+assert len(df[["ind_code", "ind_name"]].drop_duplicates()) == len(df.ind_code.unique())
+
+# #### One name per unit
+
+assert len(df[["unit_code", "unit_name"]].drop_duplicates()) == len(
+    df.unit_code.unique()
+)
+
+# #### One name per region
+
+assert len(df[["region_code", "region_name"]].drop_duplicates()) == len(
+    df.region_code.unique()
+)
+
+# ## Capture important metadata pre-pivot
+
+ind = df[["ind_code", "ind_name"]].drop_duplicates().set_index("ind_code").ind_name
+
+ind_unit_code = (
+    df[["ind_code", "unit_code"]].drop_duplicates().set_index("ind_code").unit_code
+)
+
+ind_unit_name = (
+    df[["ind_code", "unit_name"]].drop_duplicates().set_index("ind_code").unit_name
+)
+
+regions = (
+    df[["region_code", "region_name"]]
+    .drop_duplicates()
+    .set_index("region_code")
+    .region_name
+)
+
+# ## Strip back and pivot
+
+df = df.drop(["region_name", "ind_name", "unit_name", "unit_code"], axis=1)
+
+df.head()
+
+df = df.pivot(["region_code", "year"], "ind_code", "value")
+
+df.head()
+
+# ## Add metadata
+
+t = Table(df)
+
+for ind_code in t.columns:
+    meta = t[ind_code].metadata
+    meta.short_name = ind_code.lower()
```

Comment:
Instead of `ind_code.lower()` you could use `underscore(ind_code)` from `catalog.utils`.